packages/permissions/src/components/authorized/authorized.tsx

```diff
@@ -76,39 +76,32 @@ const defaultProps: Pick<Props, 'shouldMatchSomePermissions'> = {
 };
 
 const Authorized = (props: Props) => {
-  // Check first for data fences
-  if (
-    props.actualDataFences &&
-    props.demandedDataFences &&
-    props.demandedDataFences.length > 0
-  ) {
+  // if the user has no permissions and no dataFences assigned to them, they are not authorized
+  if (!props.actualDataFences && !props.actualPermissions) {
+    return <React.Fragment>{props.render(false)}</React.Fragment>;
+  }
+
+  let hasDemandedPermissions = false;
+  let hasDemandedActionRights = false;
+  let hasDemandedDataFences = false;
+
+  if (props.demandedDataFences && props.demandedDataFences.length > 0) {
     if (!props.selectDataFenceData) {
       reportErrorToSentry(
         new Error(
           `@commercetools-frontend/permissions/Authorized: Missing data fences selector "selectDataFenceData".`
         )
       );
-      return <React.Fragment>{props.render(false)}</React.Fragment>;
     }
-    return (
```

Comment:
it didnt make sense to return here because the user might meet the general permissions without meeting the dataFence permissions